k8s/overlays/coreso/coreso-databridge-claims/swe-idcc-idcf/kustomization.yaml

```diff
@@ -2,4 +2,17 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-  - swe-idcc-idcf-data-bridge-pv-claim.yaml
+  - swe-idcc-idcf-boundary-eq-data-bridge-pv-claim.yaml
+  - swe-idcc-idcf-boundary-tp-data-bridge-pv-claim.yaml
+  - swe-idcc-idcf-coreso-sv-data-bridge-pv-claim.yaml
+  - swe-idcc-idcf-crac-data-bridge-pv-claim.yaml
+  - swe-idcc-idcf-glsk-data-bridge-pv-claim.yaml
+  - swe-idcc-idcf-ree-eq-data-bridge-pv-claim.yaml
+  - swe-idcc-idcf-ree-ssh-data-bridge-pv-claim.yaml
+  - swe-idcc-idcf-ree-tp-data-bridge-pv-claim.yaml
+  - swe-idcc-idcf-ren-eq-data-bridge-pv-claim.yaml
+  - swe-idcc-idcf-ren-ssh-data-bridge-pv-claim.yaml.yaml
```

Comment:
Maybe take advantage of this PR to fix the naming of files having ".yaml" twice at the end of their name:
- swe-idcc-idcf-ren-ssh-data-bridge-pv-claim.yaml.yaml
- swe-idcc-idcf-rte-ssh-data-bridge-pv-claim.yaml.yaml